src/_scss/components/_pagination.scss

```diff
@@ -24,15 +24,15 @@
                     background-color: $color-primary-alt-lightest;
                 }
             }
-            &.active {
-                button {
+            &.pager__item_active {
+                .pager__button {
                     background-color: $color-primary-alt-dark;
                     color: $color-white;
                     border-color: $color-primary-alt-dark;
                 }
             }
-            &.disabled {
-                button {
+            &.pager__item_disabled {
```

Comment:
A more BEM-compliant approach would be to move the `disabled` modifier class to be a member of the `pager__button` element rather than its parent. This will involve moving the JSX code to set the disable modifier class on the `<button>` instead of the parent `<li>`.

Reply:
A similar, but opposite idea applies for setting the active page.

Reply:
Moved `disabled` modifier class to the button